Adds specs to Authentication Button and process (selection page) + images (#798)
italia/eidas-it-wallet-docs: docs/en/brand-identity.rst

.. "included" file, so we start with '-' title level

Brand Identity
===============

> The IT-Wallet Brand Identity defines the system's personality and it's expressed through a set of standardized elements, including its naming and Visual Identity. It strengthens and promotes the Trust Model at the basis of the entire ecosystem, that is to say the framework that legitimizes the Actors' participation in the IT-Wallet System and ensures the security and integrity of its processes.

Author: ```suggestion
The IT-Wallet Brand Identity defines the way the IT-Wallet system feels and interacts with Users, through a set of standardized elements such as its naming and Visual Identity. It strengthens a Trust Model at the basis of the entire ecosystem, introducing visual system's trustworthiness.
```

Author: this must be brougth also in the italian translation

Reviewer: ```suggestion
The IT-Wallet Brand Identity defines the system's personality and it's expressed through a set of standardized elements, such as its naming and Visual Identity. It strengthens and promotes a Trust Model at the basis of the entire ecosystem, introducing recognizable visual elements.
```

as suggested in the italian translation
